# HG changeset patch
# User Thomas Weber <thomas.weber.mail@gmail.com>
# Date 1221162817 14400
#      Thu Sep 11 15:53:37 2008 -0400
# Node ID da2fbd22d67240470fca097d6a80a867bc529537
# Parent  e0a6e54f5a19b43362dbd09dc0e8b904dc71853e
First sentence of a doc string should read like a command

diff --git a/src/DLD-FUNCTIONS/typecast.cc b/src/DLD-FUNCTIONS/typecast.cc
--- a/src/DLD-FUNCTIONS/typecast.cc
+++ b/src/DLD-FUNCTIONS/typecast.cc
@@ -126,17 +126,17 @@ typecast (const T& x, std::string type)
     }
 
   return retval;
 }
 
 DEFUN_DLD (typecast, args, ,
   " -*- texinfo -*-\n\
 @deftypefn {Loadable Function} {} typecast (@var{x}, @var{type})\n\
-Converts from one datatype to another without changing the underlying\n\
+Convert from one datatype to another without changing the underlying\n\
 data. The argument @var{type} defines the type of the return argument\n\
 and must be one of 'uint8', 'uint16', 'uint32', 'uint64', 'int8', 'int16',\n\
 'int32', 'int64', 'single' or 'double'.\n\
 \n\
 An example of the use of typecast on a little-endian machine is\n\
 \n\
 @example\n\
 @group\n\
